vita3k/renderer/src/gl/draw.cpp

```diff
@@ -95,15 +92,11 @@ void draw(GLState &renderer, GLContext &context, GxmContextState &state, const F
     for (auto &vertex_uniform : context.vertex_set_requests) {
         gl::set_uniform(program_id, vertex_program_gxp, vertex_gl_statics, mem, vertex_uniform.parameter, vertex_uniform.data,
             log_uniforms);
-
-        delete vertex_uniform.data;
```

Comment:
On Discord I have already prove this to be valid. Though currently the data passed in command buffers doesn't allow a vector to be stored (since we only have uint8_t bufffer which size of 0x80 for each command buffer to store data). Should be same with stream data and fragment data, until I can rework to use vector.